src/components/atoms/CodeEditor.tsx

```diff
@@ -203,6 +211,23 @@ class CodeEditor extends React.Component<CodeEditorProps> {
     return [localSearchShortcut, localReplaceShortcut]
   }
 
+  getCurrentDateKeymaps(): string[] {
+    let insertLocaleDateShortcut = this.props.getCustomKeymap(
+      'insertLocaleDateString'
+    )
+    let insertDateAndTimeShortcut = this.props.getCustomKeymap(
+      'insertDateAndTimeString'
+    )
+    if (insertLocaleDateShortcut == null) {
+      insertLocaleDateShortcut =
+        osName === 'macos' ? 'Shift-Cmd/' : 'Shift-Ctrl-/'
+    }
+    if (insertDateAndTimeShortcut == null) {
+      insertDateAndTimeShortcut = osName === 'macos' ? 'Cmd-Alt/' : 'Ctrl-Alt-/'
```

Comment:
`Cmd-Alt-/`

Reply:
True, mistake, it will never happen as a shortcut will not be null, but will fix (tried it once with it being null, but probably missed to recheck on refactoring